src/Mod/TechDraw/App/DrawLeaderLine.cpp

```diff
@@ -107,20 +107,13 @@ void DrawLeaderLine::onChanged(const App::Property* prop)
 
 short DrawLeaderLine::mustExecute() const
 {
-    bool result = 0;
-    if (!isRestoring()) {
-        result =  (LeaderParent.isTouched());          //Property changed
-    }
-    if (result) {
-        return result;
+    if (!isRestoring() && LeaderParent.isTouched()) {
+        return true;  // Property changed
     }
 
     const App::DocumentObject* docObj = getBaseObject();
-    if (docObj) {
-        result = docObj->isTouched();                 //object property points to is touched
-    }
-    if (result) {
-        return result;
+    if (docObj && docObj->isTouched()) {
+        return true;  // Object property points to is touched
     }

```

Comment:
This script is probably the most weird I ever worked on. Here is just one example